public/results/index.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Seattle Flu Study</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../style.css">
+  <link rel="shortcut icon" href="../favicon.ico">
+</head>
+
+<body>
+  <header>
+    <nav class="home">
+        <menu>
+          <li>
+            <a href="/">Home</a>
+          </li>
+          <li>
+            <a href="/uw">Husky Coronavirus Testing</a>
+          </li>
+          <li>
+            <a href="https://scanpublichealth.org">SCAN</a>
+          </li>
+          <li>
+            <a href="/results">Results</a>
+          </li>
+        </menu>
+      </nav>
+    <h1>Seattle Flu Study</h1>
+    <img alt="Image with text that says 'powered by the' plus the Seattle Flu Study logo"
+      src="../images/logo_SFS.svg">
+  </header>
+
```

Comment:
I think we want to include the text that is currently on https://scanpublichealth.org/results , just replace "SCAN" with "Seattle Flu Study"?

Reply:
I actually just added this landing page as an afterthought after deciding not to make /results redirect to SecureLink. I'm awaiting a language revision from Misja for the new SFS resuts page.

Reply:
The language has been revised in a new commit.

Reply:
Thanks for updating!